modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/metal/js/components/Calculator/Calculator.soy

```diff
@@ -0,0 +1,144 @@
+{namespace Calculator}
+
+{template .render}
+	{@param index: int}
+	{@param options: list<?>}
+	{@param optionsRepeatable: list<?>}
+	{@param calculatorOptions: list<?>}
+	{@param? disableCalculatorField: bool}
+	{@param strings: [
+		addField: string,
+		theExpressionWillBeDisplayedHere: string
+		]
```

Comment:
This code indentation looks weird, can you double check if it's parsing on the check_sf?